website/src/pages/use-cases/vulnerabilities.tsx

```diff
@@ -114,7 +114,7 @@ const quoteCarouselItems = [
         header: 'Nutanix fixed Log4j in days',
         quote: 'The more we dug, the more we realized [Log4Shell] was everywhere and nowhere at the same time… Sourcegraph was the right product at the right time.',
         by: 'Jon Kohler, Technical Director of Solution Engineering at Nutanix',
-        logoImage: '/external-logos/nutanix-logo.svg',
+        logoImage: '/external-logos/nutanix-color-logo.svg',
```

Comment:
Let's make a note to integrate the updated logo files in the blockquote and carousel configs in the open Use Case PRs (Code Reuse, Code Health, and Incident Response).

Reply:
All set, thanks Tim! I may go back and update the logos and references to only reference the colour logo and then make use of the black and white filter that @st0nebraker mentioned in [her comment](https://github.com/sourcegraph/about/pull/5276#discussion_r855596583).